roles/ansible-test-splitter/files/list_changed_targets.py

```diff
@@ -48,6 +49,22 @@
     help="ansible release version to test for each jobs",
 )
 
+parser.add_argument(
+    "--pull-request",
+    dest="pull_request",
+    required=True,
```

Comment:
Could we have `required=False` by default. This will simplify the troubleshooting when we run the command locally.